CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 ## Code Contribution guidelines
 ### Dev Setup and debugging help
-Read the [FAQ on the Wiki](https://github.com/vmware/docker-volume-vsphere/wiki#faq)
+Read the [FAQ on the Wiki](https://github.com/vmware/vsphere-storage-for-docker/wiki#faq)
```

Comment:
I know this is not part of PR but url shows name as "VMware Docker Volume Driver for vSphere"